grunt_tasks/plugin.js

```diff
@@ -369,16 +370,16 @@ module.exports = function (grunt) {
                                 [`girder_plugins/${plugin}/node`]: pluginNodeDir,
                                 [`girder_plugins/${plugin}`]: webClient
```

Comment:
It went in Dec 9, 2016 in #1664, and only the latter alias is documented in https://github.com/girder/girder/blob/master/docs/plugin-development.rst#extending-the-client-side-application, but I am also ok with leaving it.